src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -85,6 +89,8 @@ export default class ArcanaTracking extends Module {
 		sealState: CLEAN_SEAL_STATE,
 		sleeveState: SleeveType.NOTHING,
 	}]
+
+	private lastDrawnBuff: BuffEvent | undefined = undefined
```

Comment:
nit; below is equivalent
```suggestion
	private lastDrawnBuff?: BuffEvent
```